data/json/items/gun/shot.json

```
@@ -766,4 +766,32 @@
     "flags": [ "RELOAD_ONE" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 6 } } ]
   }
+  {
+    "id": "makeshift_shotgun",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": { "str": "makeshift shotgun" },
+    "description": "A crude shotgun, composed of two thick steel pipes, an end cap and a nail. The lack of sights make this weapon only useful at point-blank range.",
+    "weight": "3629 g",
+    "volume": "2264 ml",
+    "looks_like": "remington_870",
```

```suggestion
    "looks_like": "pipe",
```
If it doesn't have a stock, it's going to look like a pipe.